tsdb/chunks/head_chunks_other.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020 The Prometheus Authors
```

Comment:
do you even need the `_other` suffix? Should it default to all OSes without this suffix?

Reply:
without this suffix it means the original head_chunks.go file. In that case, do we set it to 0 as default and an `init()` method in windows file?

Reply:
> Should it default to all OSes without this suffix?

With `// +build !windows` in that file, it will be all non windows OS

Reply:
```
$ cat ./vendor/golang.org/x/net/ipv4/control_unix.go                                 
// Copyright 2012 The Go Authors. All rights reserved.               
// Use of this source code is governed by a BSD-style                
// license that can be found in the LICENSE file.                    
                                                                                                                                           
// +build aix darwin dragonfly freebsd linux netbsd openbsd solaris      
```
Should we use _unix.go?

Reply:
I think it's just easier to say `!windows` instead of listing everything.

Reply:
I followed the naming from here https://github.com/prometheus/prometheus/tree/master/tsdb/fileutil
where we have some OS mentioned and `_other` for all others

Reply:
We also have a case with _unix.go in that directory

Reply:
As the (known) issue is only with windows, I would prefer to explicitly have windows and !windows. Is there any gain mentioning all explicitly?

Reply:
Another way could be get rid of `_others`, have the variable in main file, and have an `init()` function in `_windows`.

Reply:
No gain, no. I was just pointing out how upstream go does it but at the end it is your call.